contracts/src/v0.8/ccip/libraries/Internal.sol

```diff
@@ -278,6 +278,9 @@ library Internal {
   // bytes4(keccak256("CCIP ChainFamilySelector SVM"));
   bytes4 public constant CHAIN_FAMILY_SELECTOR_SVM = 0x1e10bdc4;
 
+  // bytes4(keccak256("CCIP ChainFamilySelector APTOS"));
+  bytes4 public constant CHAIN_FAMILY_SELECTOR_APTOS = 0xac77ffec;
```

Comment:
Do we usually have tests for these that enforce that the bytes4 value == `bytes4(keccak256("CCIP ChainFamilySelector APTOS"))`?

Reply:
No, we have a PR review. I don't love tests that assert a constant is a constant

Reply:
Sounds good, hope others checked, I didn't 😛